vcpkg/maintainers/functions/vcpkg_make_configure.md

```diff
@@ -0,0 +1,164 @@
+---
+title: vcpkg_make_configure
+description: Use vcpkg_make_configure to configure a Make-based project.
+ms.date: 01/30/2024
+---
+# vcpkg_make_configure
+
+Configure a Make-based project.
+
+## Usage
+        
+```cmake
+vcpkg_make_configure(
+    [AUTOCONFIG]
+    [COPY_SOURCE]
+    [DISABLE_MSVC_WRAPPERS]
+    [DISABLE_CPPFLAGS]
+    [DISABLE_DEFAULT_OPTIONS]
+    [DISABLE_MSVC_FLAG_ESCAPING]
+    [USE_RESPONSE_FILES]
+    [SOURCE_PATH <source-path>]
+    [OPTIONS <configure-setting>...]
+    [OPTIONS_RELEASE <configure-setting>...]
+    [OPTIONS_DEBUG <configure-setting>...]
+    [PRE_CONFIGURE_CMAKE_COMMANDS <cmake-command>...]
+    [LANGUAGES <language>...]
+)
+```
+
+To use this function, you must depend on the helper port `vcpkg-make`:
+
+```json
+"dependencies": [
+  {
+    "name": "vcpkg-make",
+    "host": true
+  }
+]
+```
+
+## Parameters
+
+### AUTOCONFIG
+
+Runs autoreconf when set.
+
+### COPY_SOURCE
+
+If specified, the source directory will be copied to the build directory before the configuration process begins. Useful for projects that modify the source tree during configuration or build.
+
+### DISABLE_MSVC_WRAPPERS
+
+By default, `vcpkg_make_configure` supplies wrappers that translate and reorder 'unix / gcc-style' flags into 'msvc-style' flags. This accommodates build systems that assume unix or gcc conventions.
+
+Setting `DISABLE_MSVC_WRAPPERS` removes these wrappers and allows the underlying make build system to supply 'msvc-style' flags directly without translation. When this option is enabled, the `cl` and `windres` tools will directly interpret command-line flags without any modifications.
+
+### DISABLE_MSVC_FLAG_ESCAPING
+
+By default, escape characters (e.g., -Xcompiler, -Xlinker) are added before compiler and linker flags when using MSVC. These escape characters are intended for use by the wrappers and libtool to protect flags that might contain spaces, quotes, or other special characters from being misinterpreted.
+When you set `NO_MSVC_FeLAG_ESCAPING`, you tell vcpkg not to perform this automatic escaping. This can be useful in situations where:
```

Comment:
```suggestion
Setting `DISABLE_MSVC_FLAG_ESCAPING` disables this behavior. This can be useful in situations where:
```

Is it `NO_` or `DISABLE_`, or both?

Reply:
Just `DISABLE_`